docs/installation.md

```diff
@@ -123,13 +123,8 @@ Where `[GHE]` is our GitHub Enterprise URL.
 
 #### Set up your environment
 
-If this is the first time you're setting up the project, run the following
-script to copy `.env_SAMPLE` to `.env`, export your environment variables,
-and activate your virtualenv for the first time.
-
-```bash
-source load-env.sh
-```
+If this is your first time setting up the project, copy `.env_SAMPLE` to
+`.env`, then `source .env` and activate your virtualenv.
```

Comment:
For people who are less technical, it might be nice to include the explicit steps:

```sh
$ cp .env_SAMPLE .env
$ source .env
$ workon cfgov-refresh
```

Reply:
I don't know; that's why I asked :)

The man page says "Preserves structure and attributes of files but not directory structure," which I don't totally understand.

Reply:
my guess is that it was was someone being overly-cautious, but I'll leave the `-a` in for now just to be safe